plaso/cli/psort_tool.py

```diff
@@ -239,9 +240,16 @@ def _ParseProcessingOptions(self, options):
 
     if worker_memory_limit and worker_memory_limit < 0:
       raise errors.BadConfigOption(
-          'Invalid worker memory limit value cannot be negative.')
+          'Invalid worker memory limit value cannot be less than 0.')
```

Comment:
Please include the invalid value in the error message here. For example: "Invalid memory limit -1. Limit cannot be less than 0."

Reply:
done